alex/applications/PublicTransportInfoCS/data/convert_idos_stops.py

```diff
@@ -28,7 +28,7 @@
 import sys
 import re
 from add_cities_to_stops import load_list, get_city_for_stop
-from alex.utils.various import remove_dups_stable
+from alex.utils.various import list_remove_duplicates_keep_ordering
```

Comment:
The problem is that I had to study the code of the method to understand what is it doing - it was not apparent to me on the first sight it is working on a list. So how about list_remove_dups_stable ?